diff --git a/server/settings/requireClientPassword.ini b/server/settings/requireClientPassword.ini
index c227083..56a6051 100644
--- a/server/settings/requireClientPassword.ini
+++ b/server/settings/requireClientPassword.ini
@@ -1 +1 @@
-0
\ No newline at end of file
+1
\ No newline at end of file
diff --git a/server/settings/requireTicketServerCheck.ini b/server/settings/requireTicketServerCheck.ini
index 56a6051..c227083 100644
--- a/server/settings/requireTicketServerCheck.ini
+++ b/server/settings/requireTicketServerCheck.ini
@@ -1 +1 @@
-1
\ No newline at end of file
+0
\ No newline at end of file
diff --git a/server/settings/useLineageServer.ini b/server/settings/useLineageServer.ini
index 56a6051..c227083 100644
--- a/server/settings/useLineageServer.ini
+++ b/server/settings/useLineageServer.ini
@@ -1 +1 @@
-1
\ No newline at end of file
+0
\ No newline at end of file
diff --git a/server/settings/useStatsServer.ini b/server/settings/useStatsServer.ini
index 56a6051..c227083 100644
--- a/server/settings/useStatsServer.ini
+++ b/server/settings/useStatsServer.ini
@@ -1 +1 @@
-1
\ No newline at end of file
+0
\ No newline at end of file
